reference.py

```diff
@@ -330,7 +322,14 @@ def scanning(b_scan: ECKey, B_spend: ECPubKey, A_sum: ECPubKey, input_hash: byte
                 assert pub_key.verify_schnorr(sig, msg), f"Invalid signature for {pub_key}"
                 output["signature"] = sig.hex()
 
-            # Check if the found output public keys match the expected output public keys
-            assert add_to_wallet == expected["outputs"], "Receiving test failed"
+            # Note: order doesn't matter for creating/finding the outputs. However, different orderings of the recipient addresses
```

Comment:
I think this comment here is irrelevant for the recipient part, no?

Reply:
Ah, good call. This is a leftover from when I did have all possible output sets for the recipient. But now for the recipient there is just one of the possible output sets generated by the sender, so this comment is indeed irrelevant.